modules/renter/contractor/contractor_test.go

```diff
@@ -459,7 +459,7 @@ func TestIntegrationSetAllowance(t *testing.T) {
 		t.Fatal(err)
 	}
 	c.mu.Lock()
-	clen := c.contracts.Len()
+	clen := c.staticContracts.Len()
 	c.mu.Unlock()
```

Comment:
it looks like we lock and then unlock just to get the length? Should be good to drop that lock and unlock, right?